prusti-viper/src/ide/vsc_span.rs

```diff
@@ -0,0 +1,41 @@
+use prusti_rustc_interface::span::{source_map::SourceMap, Span};
+use serde::Serialize;
+
+/// a representation of spans that is more usable with VSCode.
+#[derive(Serialize, Clone)]
+pub struct VscSpan {
+    column_end: usize,
+    column_start: usize,
+    line_end: usize,
+    line_start: usize,
+    file_name: String,
+    is_primary: bool,
+    label: Option<()>,
+    expansion: Option<()>,
```

Comment:
Can you explain these two fields? Why are they not just `bool`s?

Reply:
The idea was to allow easy serialization here, and deserialization in typescript, where spans of this form are already parsed. (this is the form they have when emitting compiler messages with spans).

`label` and `expansion` are both optional fields in the type defined in prusti-assistant, so putting in a boolean would not work. Option fields with value None, when serialized, will result in no field. But since I don't use them at all I can just delete them.